### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by C-Jaewon** [tzm-tora/Stroke-Based-Scene-Text-Erasing#7]
We created train datasets using synthtext and finetuned the model. However, the predicted mask image was printed as a black image.
![i_s_12](https://user-images.githubusercontent.com/20211880/168537135-93177a42-e65e-405a-a23d-09b7894d0ffe.jpg)
![mask_t_12](https://user-images.githubusercontent.com/20211880/168537149-44427f6c-981e-4eda-88e1-b1728d5d297f.png)
![t_b_12](https://user-images.githubusercontent.com/20211880/168537157-1d1e917d-7adc-4af2-84ec-e597be1cf7fb.jpg)
train dataset
![display](https://user-images.githubusercontent.com/20211880/168538465-01ebbbf8-0821-473c-ba3e-e35c21ec979e.jpg)
test result

1. Is the mask image a binary image or rgb image?
2. If not, what is the problem?

Thanks

**Comment by tzm-tora** [comment]
hi, from the last row of the test result, it seems that the model is still working. The predicted mask should be a 3-channel greyscale image, whose values are in (0, 1). The pixels of text are close to 0, and the pixels of background are close to 1.

**Comment by tzm-tora** [comment]
hi, from the last row of the test result, it seems that the model is still working. The predicted mask should be a 3-channel greyscale image, whose values are in (0, 1). The pixels of text are close to 0, and the pixels of background are close to 1. I am not sure if you multiply 255 when you display the mask image.

**Comment by C-Jaewon** [comment]
It was a channel problem with the mask_t image.

Thanks
